fabric-content-registries-v0/src/main/java/org/quiltmc/quilted_fabric_api/impl/content/registry/plugin/QuiltedFabricContentRegistriesMixinPlugin.java

```diff
@@ -0,0 +1,45 @@
+package org.quiltmc.quilted_fabric_api.impl.content.registry.plugin;
+
+import java.util.List;
+import java.util.Set;
+
+import org.objectweb.asm.tree.ClassNode;
+import org.quiltmc.loader.api.QuiltLoader;
+import org.spongepowered.asm.mixin.extensibility.IMixinConfigPlugin;
+import org.spongepowered.asm.mixin.extensibility.IMixinInfo;
+
+public class QuiltedFabricContentRegistriesMixinPlugin implements IMixinConfigPlugin {
+	@Override
+	public void onLoad(String mixinPackage) {
+	}
+
+	@Override
+	public String getRefMapperConfig() {
+		return null;
+	}
+
+	@Override
+	public boolean shouldApplyMixin(String targetClassName, String mixinClassName) {
+		return !QuiltLoader.isDevelopmentEnvironment() && !Boolean.getBoolean("quilted_fabric_api.disable_rea_fix");
+	}
+
+	@Override
+	public void acceptTargets(Set<String> myTargets, Set<String> otherTargets) {
+
+	}
```

Comment:
```suggestion
	public void acceptTargets(Set<String> myTargets, Set<String> otherTargets) {
	}
```
I'd suggest `{}`s, but I doubt that Fabric's checkstyle (which is much stricter) will accept it